default.nix

```diff
@@ -29,5 +29,5 @@ pkgs.stdenv.mkDerivation {
 
   configurePhase = "sh ./configure.sh";
 
-  installFlags = "COQLIB=$(out)/lib/coq/${coq.coq-version}/";
+  installFlags = "COQMF_COQLIB=$(out)/lib/coq/${coq.coq-version}/";
```

Comment:
It looks like this change is not compatible with Coq 8.6. Should I revert it or should we drop 8.6 compatibility?

Reply:
Drop the 8.6 compatibility.

Reply:
Done.